views/task-list.html

```diff
@@ -2,7 +2,7 @@
 {% from "components/navigation-tabs.html" import navigationTabs with context %}
 {% set pageHeader = i18n.taskList.header %}
 {% block pageTitle %}
-    {{ i18n.taskList.header }} - {{ i18n.general.serviceTitle }} - {{ i18n.general.govUK }}
+    {{ i18n.taskList.header }} - {{ i18n.provideEvidenceTab.tabHeader }} - {{ i18n.general.govUK }}
```

Comment:
Was it requirement to change service title to tab header?

Reply:
Yeah, it's not on the ticket but Hemingway asked if I could add it in. It's so we get a different page title for each tab (status, hearing, etc).